src/types/index.ts

```diff
@@ -4,7 +4,6 @@ export * from './classes'
 export * from './ColorMode'
 export * from './css'
 export * from './dates'
-export * from '../models/api/LogsRequestFilter'
```

Comment:
What is this change for?

Reply:
I asked him to fix duplicate exports. I recognize `LogsRequestFilter`, which I moved out of types and into models/api. Looks like vscode automatically "fixed" this import